Tests/Functional/Form/CheckboxUrlLabelFormTypeTest.php

```diff
@@ -24,9 +25,8 @@ public function setUp()
 
     public function testFormTwigTemplate()
     {
-        $twigExtension = $this->getContainer()->get('twig');
-        $twigExtension->initRuntime();
```

Comment:
this method was deprecated and is no longer supported. as far as i can tell we don't need to call it, right?